src/backendMock/db.json

```diff
@@ -1,77 +1,48 @@
 {
-  "articles": [
```

Comment:
i changed the whole naming of articles to products based on your recommendation, i will stick with product for future work